packages/admin/resources/views/components/layouts/app/sidebar/item.blade.php

```diff
@@ -4,11 +4,18 @@
     'badge' => null,
     'badgeColor' => null,
     'icon',
+    'visible' => null,
+    'hidden' => null,
```

Comment:
I don't think any views need to be changed to accommodate this. If a navigation item is hidden, then it should not be in the array of items in the first place

Reply:
So these changes should be removed and replaced with changes in the code that returns the navigation structure. Test should also be updated